src/zabapgit_objects_impl.prog.abap

```diff
@@ -478,6 +478,8 @@ CLASS lcl_objects IMPLEMENTATION.
         zcx_abapgit_exception=>raise( 'cancelled' ).
       ENDIF.
 
+      zcl_abapgit_default_task=>get_instance( )->set( lv_package ).
+
```

Comment:
perhaps move this outside the loop, so get and set is at the same level? I dont think it matters which package in the hierarchy is chosen, all should have the same transportation settings.

Reply:
I moved the block to the begin of the method. As method grows and the new parts of this PR are at the begin and the end I'd suggest to extract the middle part into a separat method. What do you think?